src/source/PeerConnection/SessionDescription.c

```diff
@@ -582,6 +610,42 @@ STATUS populateSessionDescriptionDataChannel(PKvsPeerConnection pKvsPeerConnecti
     return retStatus;
 }
 
+BOOL isPresentInRemote(PKvsRtpTransceiver pKvsRtpTransceiver, PSessionDescription pRemoteSessionDescription) {
+
+    PCHAR remoteAttributeValue, end;
+    UINT32 remoteTokenLen;
+
+    MEDIA_STREAM_TRACK_KIND localTrackKind = pKvsRtpTransceiver->sender.track.kind;
+
+    for (int i = 0; i < pRemoteSessionDescription->mediaCount; i++) {
```

Comment:
Same question regarding local variables. In this case, it's for the aesthetics and to make the code look the same